cypress/cypress.config.js

```diff
@@ -18,13 +18,20 @@ module.exports = defineConfig({
     runMode: 2,
   },
   e2e: {
+    setupNodeEvents(on, config) {
```

Comment:
Did we need 2 cypress configs?

Reply:
@sirugh Please let me know if we still need to update this setup -
1. GQL endpoint we use in some tests for calls to resolve example -  https://github.com/hlxsites/aem-boilerplate-commerce/blob/b6686e0002bded77bcc015bc5e444e1ef643f329/cypress/src/tests/e2eTests/verifyGuestUserCheckout.spec.js#L86 
2. This different config flow I used per cypress docs - https://docs.cypress.io/api/node-events/configuration-api#Switch-between-multiple-configuration-files

Reply:
Regarding point 2, the flow from cypress defines json files which are not extensible, but you have `.js` files so you could make it simpler. Was my example not clear? You would have a base config, then you would have saas and paas configs which import/overwrite:
```
//cypress.base.js
export base = {
   defaultCommandTimeout: 60000,
  //etc
}

// cpyress.saas.js
import base from cypress.base.js
export default {
  ...base
  env: {
    ...base.env,
    graphqlEndpoint: SAAS_ENDPOINT,
    // etc
}

// cypress.paas.js
import base from cypress.base.js
export default {
  ...base
  env: {
    ...base.env,
    graphqlEndpoint: PAAS_ENDPOINT,
    // etc
}
```